### PR TITLE
Stratonovich SRK methods due to Roessler

### PR DESCRIPTION
**Pull request by frankschae**
Adds RS1 and RS2 [1]  for the weak approximation in the Stratonovich sense from issue #182 .
Tests for  diagonal and non-commuting noise are taken from [1]. 

[[1]Roessler's paper]( https://link.springer.com/content/pdf/10.1007/s10543-007-0130-3.pdf)

### CODE REVIEW

File: src/StochasticDiffEq.jl [hunk 1]
```diff
@@ -151,7 +152,6 @@ module StochasticDiffEq
   export solve, init, solve!, step!
 
   #Misc Tools
-  export checkSRIOrder, checkSRAOrder,  checkRIOrder, constructSRIW1, constructSRA1, constructDRI1, constructRI1, constructRI3, constructRI5, constructRI6,
-        constructRDI1WM, constructRDI2WM, constructRDI3WM, constructRDI4WM
+  export checkSRIOrder, checkSRAOrder,  checkRIOrder, checkRSOrder, constructSRIW1, constructSRA1, constructDRI1, constructRI1, constructRI3, constructRI5, constructRI6, constructRDI1WM, constructRDI2WM, constructRDI3WM, constructRDI4WM, constructRS1, constructRS2
```
Comment (ChrisRackauckas) [inline src/StochasticDiffEq.jl:155]: whatta line

Comment (frankschae) [reply src/StochasticDiffEq.jl]: I added some line breaks 👍

**Comment by frankschae** [comment]
Had to create a 4th group of weak tests -- they are too long to fit together with other tests in the 50 min Travis window. 

Once the adaptive schemes from PR #332 are merged, I'll start moving also weak tests to the GPU.  (The CI GPU tests however still don't seem to start. They freeze at pending.)

**Comment by ChrisRackauckas** [comment]
https://gitlab.com/JuliaGPU/StochasticDiffEq.jl/-/jobs/600755893 they are fine, just the section is empty for them right now.

**Comment by ChrisRackauckas** [comment]
We need to move to other GPU CI machines first though. I think Tim might be getting a bit frustrated with us killed machines in the OrdinaryDiffEq one haha

**Pull request merged**